### PR TITLE
Fs jul19 konstantin b

### PR DESCRIPTION
**Pull request by ghost**
layout_style-it-up solution by K Burnazaki

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#footer {
+  width: 300px;
+}
+
+.important,
+.interesting,
+.surprising,
+#header,
+#footer {
+  height: 100px;
```
Comment (YuriiHoliuk) [inline src/style.css]: write it in the above block

**Review by YuriiHoliuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 1 above)
Comment (burnazaki) [reply src/style.css]: done

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#footer {
+  width: 300px;
+  height: 100px;
+}
+
+.important:hover,
+.interesting:hover,
+.surprising:hover,
+#header:hover,
+#footer:hover {
```
Comment (YuriiHoliuk) [inline src/style.css]: but you should delete this declaration

**Review by YuriiHoliuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (burnazaki) [reply src/style.css]: This declaration makes height change when hovered over. If I delete this declaration the solution will not be in compliance with the requirements. Or I misunderstood smth?

Comment (YuriiHoliuk) [reply src/style.css]: Oh, sorry i missed hover

**Review by YuriiHoliuk — APPROVED** [review]

**Pull request closed without merging**